docs/content/concepts/automation/schedules/running-assets.mdx

```diff
@@ -0,0 +1,263 @@
+
+```python file=concepts/partitions_schedules_sensors/schedules/basic_asset_schedule.py startafter=start_assets endbefore=end_assets
+from dagster import asset
+
+
+@asset(group_name="ecommerce_assets")
+def orders_asset():
+    return 1
+
+
+@asset(group_name="ecommerce_assets")
+def users_asset():
+    return 2
+```
+
+To create a job that materializes the assets in this group, we'll use <PyObject object="define_asset_job" />.
+
+```python file=concepts/partitions_schedules_sensors/schedules/basic_asset_schedule.py startafter=start_job endbefore=end_job
+from dagster import AssetSelection, define_asset_job
+
+
+ecommerce_asset_job = define_asset_job(
+    "ecommerce_asset_job", AssetSelection.groups("ecommerce_assets")
+)
+```
+
+To do this, we:
+
+1. Imported <PyObject object="AssetSelection" /> and <PyObject object="define_asset_job" />
+2. Used <PyObject object="define_asset_job" /> to construct the job and name it `ecommerce_asset_job`
+3. Used <PyObject object="AssetSelection" /> to select all assets in the `ecommerce_assets` group. Only these assets will be materialized when the job runs.
+
+Refer to the [Asset jobs documentation](/concepts/assets/asset-jobs) for more info and examples.
+
+---
+
+## Step 2: Define the schedule
+
+Next, we'll use <PyObject object="ScheduleDefinition" /> to construct the schedule and attach it to the job we created in [Step 1](#step-1-create-the-job).
+
+```python file=concepts/partitions_schedules_sensors/schedules/basic_asset_schedule.py startafter=start_schedule endbefore=end_schedule
+from dagster import ScheduleDefinition
+
+
+ecommerce_schedule = ScheduleDefinition(
+    job=ecommerce_asset_job,
+    cron_schedule="15 5 * * 1-5",
+    default_status=DefaultScheduleStatus.RUNNING,
+)
+```
+
+In this example, we:
+
+1. Imported <PyObject object="ScheduleDefinition" /> and the `ecommerce_asset_job`
+2. Uused <PyObject object="ScheduleDefinition" /> to create a schedule that:
+
+   - Is attached to the `ecommerce_assets_job` job
+   - Has a cron expression of `15 5 * * 1-5`, which translates to `Every Monday through Friday of every month at 5:15AM`
+   - Is turned on by default (`default_status`). We'll discuss this more in [Step 4](#step-4-turn-the-schedule-on).
+
+---
+
+## Step 3: Update the Definitions object
+
+Next, we'll update our project's <PyObject object="Definitions" /> object to include our new job and schedule. This ensures the job and schedule are available to Dagster processes like the Dagster UI.
+
+```python file=concepts/partitions_schedules_sensors/schedules/basic_asset_schedule.py startafter=start_definitions endbefore=end_definitions
+from dagster import Definitions
+
+
+defs = Definitions(
+    assets=[orders_asset, users_asset],
+    jobs=[ecommerce_asset_job],
+    schedules=[ecommerce_schedule],
+)
+```
+
+At this point, our code looks like this:
+
+```python file=concepts/partitions_schedules_sensors/schedules/basic_asset_schedule.py lines=3-10,14-23,30-34,41-47,54-60
```

Comment:
I added a new property `hidecomments` so you can do `hidecomments=true` in my PR. Even if we don't merge my docs changes, I can create a PR for just the `python file...` enhancement so you don't have to specify every single line next time.

Reply:
omg. That would be amazing.

But this also removes the individual imports from the iterative portions of the code... if we could specify removing only specific lines, that would also be wonderful.